loader/loader.go

```diff
@@ -237,15 +243,20 @@ func (l *loader) GetRoleBindingFromReleaseStatus(ctx context.Context, cli client
 	return roleBinding, nil
 }
 
-// GetReleasePipelineRun returns the Release PipelineRun referenced by the given Release or nil if it's not found. In the case
-// the List operation fails, an error will be returned.
-func (l *loader) GetReleasePipelineRun(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*tektonv1.PipelineRun, error) {
+// GetReleasePipelineRun returns the Release PipelineRun of the specified type referenced by the given Release
+// or nil if it's not found. In the case the List operation fails, an error will be returned.
+func (l *loader) GetReleasePipelineRun(ctx context.Context, cli client.Client, release *v1alpha1.Release, pipelineType string) (*tektonv1.PipelineRun, error) {
```

Comment:
We could think about using an enum for the pipelines types as is less error prone. Maybe create a ticket for the future?

Reply:
https://issues.redhat.com/browse/RELEASE-1180